lib/builder.ts

```diff
@@ -10,7 +10,6 @@ import BuildError from './errors/build';
 import CancelationRequest from './cancelation-request';
 import Cancelation from './errors/cancelation';
 import filterMap from './utils/filter-map';
-import promiseFinally from 'promise.prototype.finally';
```

Comment:
Removing this, and using async/await is a breaking change, since node 8 does not support `finally`.

I am fine with us releasing a new major that drops node 8, but it will require a longer release cycle to be available in ember-cli. 

If the issue can be fixed without making this change, it can be released as a patch version

Reply:
It's not a breaking change. It's not being used anymore because with async/await try/finally works as expected in node 8. I was able to rewrite all the promises that were using `promiseFinally` to use await within a try/finally block instead.

Reply:
Using async/await is not a breaking change. The node support is already node 8+.

Reply:
You are correct using async/wait is not a breaking change, but returning a promise from `build()` that doesn't have `finally` is a breaking change. That being said, I just reading `promise.prototype.finally` code, and it does not return a promise with a `finally` method. 

So you are correct, this wont be a breaking change.